includes/class-wc-beta-tester-admin-assets.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Beta Tester Admin Assets class
+ *
+ * @package WC_Beta_Tester
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Beta_Tester_Admin_Assets Class.
+ */
+class WC_Beta_Tester_Admin_Assets {
+
+	/**
+	 * Hook in tabs.
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_scripts' ) );
+	}
+
+	/**
+	 * Enqueue scripts.
+	 */
+	public function admin_scripts() {
+		$suffix       = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		// Need admin styles for the modal.
+		wp_enqueue_style( 'woocommerce_admin_styles' );
+
+		wp_register_script( 'wc-beta-tester-version-info', WC_Beta_Tester::instance()->plugin_url() . '/assets/js/version-information' . $suffix . '.js', array( 'wc-backbone-modal' ), WC_BETA_TESTER_VERSION );
+
+		// TODO: Get version programmatically.
+		$version = '3.4.0';
```

Comment:
This needs to use `WC_BETA_TESTER_VERSION`

Reply:
Are you sure? This parameter is used to get information about the WooCommerce version

Reply:
Are you suggesting to use `WC_COMMERCE`? I think it's a good idea but not sure if there's a case it could cause issues?